bin/hdfcoinc/pycbc_multiifo_add_statmap

```diff
@@ -11,6 +11,17 @@ import pycbc.conversions as conv
 from pycbc.events import coinc
 from ligo import segments
 
+def get_ifo_attr(fi):
```

Comment:
put 1 line of comment saying what it does, ie what the return value is supposed to be

Reply:
also, since in one case it's not an attr may want to call this something else